svc-bie-kafka/src/main/resources/application.yaml

```diff
@@ -0,0 +1,37 @@
+spring:
+  # Specify where Spring connects to RabbitMQ
+  rabbitmq:
+    host: "${RABBITMQ_PLACEHOLDERS_HOST:localhost}"
+    port: 5672
+    username: "${RABBITMQ_PLACEHOLDERS_USERNAME:guest}"
+    password: "${RABBITMQ_PLACEHOLDERS_USERPASSWORD:guest}"
+
+  # Specify where Spring connects to Kafka
+  kafka:
+    consumer:
+      group-id: "${BIE_KAFKA_PLACEHOLDERS_GROUP_ID:vro-bie-group}"
+    bootstrap-servers: "${BIE_KAFKA_PLACEHOLDERS_BROKERS:localhost:9092}"
+
+## Specify bie properties
+bie:
+  topic-map:
+    # Map of entries where the key is a kafka topic pattern (regex) to which this app will subscribe. This pattern will
+    # be prepended and appended with wildcards to allow for any prefix or suffix in the kafka topic name. The value is
```

Comment:
If I understand this correctly, this seems risky to match for `*KEY*` when we don't have control over the list of Kafka topics. For example, if BIE adds a `DONT_USE_THIS_TEST_ONLY_CONTENTION_UPDATED` topic, then VRO will receive those unintended notifications. I suggest having the key be the exact topic name.

Reply:
Ah thats a good point!

Reply:
My initial suggestion is to create a `bie.topic-prefix` property, which we can set for each LHDI env using `application-$ENV.yaml` -- see https://github.com/department-of-veterans-affairs/abd-vro/blob/0d0d5b5e0adb2561ac34e7594828d3dc03703276/app/src/main/resources/application.yml#L96-L112